src/sentry/profiles/utils.py

```diff
@@ -147,7 +156,7 @@ def parse_profile_filters(query: str) -> dict[str, str]:
     try:
         parsed_terms = parse_search_query(query)
     except ParseError as e:
-        raise InvalidSearchQuery(f"Parse error: {e.expr.name} (column {e.column():d})")
+        raise InvalidSearchQuery(f"Parse error: {e.expr.name} (column {e.column():d})")  # type: ignore[union-attr]
```

Comment:
this is pointing at an actual issue I believe -- should probably attempt to safely unpack the values instead